hack/addons/kustomize/nutanix-snapshot-csi/kustomization.yaml.tmpl

```diff
@@ -16,4 +16,4 @@ helmCharts:
   version: ${NUTANIX_SNAPSHOT_CSI_CHART_VERSION}
   includeCRDs: true
   skipTests: true
-  namespace: nutanix-system
+  namespace: kube-system
```

Comment:
The CSI Pods are failing
```
kube-system              nutanix-csi-controller-574bdd7b96-7zvzf                                    5/7     CrashLoopBackOff   17 (18s ago)    3m11s
kube-system              nutanix-csi-controller-574bdd7b96-kcngj                                    2/7     Error              11 (13s ago)    3m11s
kube-system              nutanix-csi-node-2mlh2                                                     1/3     CrashLoopBackOff   9 (5s ago)      2m15s
kube-system              nutanix-csi-node-rcdv4                                                     2/3     CrashLoopBackOff   7 (15s ago)     109s
```

with:
```
W0620 20:17:32.974602       8 client_config.go:614] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
2024-06-20T20:17:32.981Z main.go:515: [ERROR] Error getting the data from ConfigMap ntnx-init-configmap/ntnx-system : configmaps "ntnx-init-configmap" not found
F0620 20:17:32.981945       8 main.go:389] Error reading data from init configmap, err: configmaps "ntnx-init-configmap" not found
```

The namespace can be configured, see [here](https://github.com/nutanix-core/k8s-csi/blob/5dbcb6e20b58d404d26703a36cad853f125efaab/cmd/init/main.go#L74-L77) but the Helm chart does not support setting neither an extra arg or an env, see [here](https://github.com/nutanix/helm/blob/csi-v3.0.0-beta-beta1/charts/nutanix-csi-storage/templates/ntnx-csi-controller-deployment.yaml).

Reply:
This file isn't used by the way... it's only used for CRS manifest creation, but is used by the helm cm builder utility to list helm charts being used.